### DISCUSSION TITLE
Multiple DRG types

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#47] [Q&A] [answered]
There isn't just one type of DRG code. There is MS-DRG and also APR-DRG.  See: 

https://www.healthcare-economist.com/2012/06/19/what-is-the-difference-between-drgs-ap-drgs-and-apr-drgs/

**Comment by tdfow** [comment, marked as answer]
If the billing code is DRG, then I expect that MS-DRG or APR-DRG would be noted in the billing_code_type_version field.

**Reply by nmayle** [reply]
I had assumed that the version would be the version number of the specific coding system and not used to represent different coding systems.  To me, that fits better in the billing_code_type.  One could consider MS-DRG and APR-DRG the same code type since they are both systems of DRGs.

**Reply by tdfow** [reply]
For example, wouldn't this version identify the appropriate version, MS-DRG V38.0--as the billing code type, as it identifies the DRG type/version

**Reply by nmayle** [reply]
Yes, you could that, but DRG is an approach to billing and MS-DRG is a code_type.  Right?

The concern is that in many cases if one were to match data across time periods there may be different versions of the MS-DRG in use, and you will still want to be able to compare items that have the same code.  

To determine that both items are MS-DRG codes would requiring pulling apart the billing_code_type_version field and extracting the "MS-DRG" part.  To me, it seems much cleaner for the billing_code_type to be MS-DRG and the billing_code_type_version to be V38.0 or 38.0.  What is gained by both MS-DRG and APR-DRGs both being code_type=DRG and burying the coding type in the version?

**Reply by tdfow** [reply]
Based on the current schema--it seemed DRG was the only billing_code_type for DRG.  MS-DRG or APR-DRG  were not noted as valid values.   Hopefully, CMS can provide examples of how they want to handle--and either change the billing_code_type allowed values or whether they expect it to be part of the billing_code_type_version information.

**Reply by nmayle** [reply]
I agree, that as it currently stands, one would have to hack this in.  However, isn't this schema a work in progress?  Isn't the point of our comments to improve and/or correct the schema where it doesn't match the world of billing or the language of the rules?

**Reply by tdfow** [reply]
It does seem cleaner to have MS-DRG and APR-DRG to be the billing_code_type values.    Regarding the billing_code_type_version, I also feel they should specify if there are any formatting expectations/guidance for the different versions for the billing code types.

**Reply by taylorpatriciab** [reply]
I concur that it's cleaner to have MS-DRG and APR-DRG be the billing_code_type values.  Each then has versions that correlate to the annual releases.  The billing_code_type_version would capature that value.

**Reply by tdfow** [reply]
@shaselton Any CMS updates to whether the Billing Code Type Values in schema will be updated to reflect MS-DRG, APR-DRG instead of just DRG?

**Reply by shaselton-usds** [reply]
@nmayle Hopefully this was answered in over here: https://github.com/CMSgov/price-transparency-guide/discussions/9

cc @tdfow @taylorpatriciab
